code/processes/dqe.q

```diff
@@ -0,0 +1,34 @@
+\d .dqe
+
+/- default parameters
+subscribeto:@[value;`subscribeto;`];                          /-list of tables subscribed to
+subscribesyms:@[value;`subscribesyms;`];                      /-list of syms subscribed to
+schema:@[value;`schema;0b];                                   /-retrieve schema from tickerplant
+tickerplanttypes:@[value;`tickerplanttypes;`tickerplant];     /-list of tickerplant types to try and make a connection to
+replaylog:@[value;`replaylog;0b];                             /-replay the tickerplant log file
+tpconnsleepintv:@[value;`tpconnsleepintv;10];                 /-default wait time before trying to reconnect to discovery/TP
+/- end of default parameters
+
+subscribe:{[]
+  if[count s:.sub.getsubscriptionhandles[tickerplanttypes;();()!()];
```

Comment:
No indentation passed this point? :cry:
I'm all for changing this up a bit to exit early here:
```suggestion
  if[0=count s:.sub.getsubscriptionhandles[tickerplanttypes;();()!()];:()];
```
(needs the closing bracket removed on line 19)